src/table/components/head/HeadCellMenu.tsx

```diff
@@ -1,23 +1,39 @@
 import React, { useRef, useState, useMemo } from 'react';
 import Menu from '@mui/material/Menu';
-import More from '@qlik-trial/sprout/icons/react/More';
-import Search from '@qlik-trial/sprout/icons/react/Search';
+import More from '@qlik-trial/sprout/icons/More';
+import Search from '@qlik-trial/sprout/icons/Search';
 
 import { useContextSelector, TableContext } from '../../context';
 import { HeadCellMenuProps, MenuItemGroup } from '../../types';
-import { StyledMenuIconButton, NebulaListBox } from './styles';
-import { ListBoxWrapper, ListBoxWrapperRenderProps } from './ListBoxWrapper';
+import { StyledMenuIconButton } from './styles';
 import MenuItems from './MenuItems';
 
-export default function HeadCellMenu({ columnIndex, isDimension, tabIndex }: HeadCellMenuProps) {
-  const { translator } = useContextSelector(TableContext, (value) => value.baseProps);
+export default function HeadCellMenu({ column, tabIndex }: HeadCellMenuProps) {
+  const { translator, embed } = useContextSelector(TableContext, (value) => value.baseProps);
   const [openMenuDropdown, setOpenMenuDropdown] = useState(false);
   const [openListboxDropdown, setOpenListboxDropdown] = useState(false);
-  const anchorRef = useRef<HTMLButtonElement>(null);
+  const anchorRef = useRef<HTMLDivElement>(null);
+  const listboxRef = useRef<HTMLDivElement>(null);
+  const showListboxMenu = column.isDim && !column.isMasterDim;
```

Comment:
this isn't really what it is. This is if we should show the search menu item (at this point). So should be something like `showSearchMenuItem`